rtsmith.cpp

```diff
@@ -166,7 +166,7 @@ std::optional<RtSmithResult> RtSmith::generateConfig(const std::string &program,
 
 std::optional<RtSmithResult> RtSmith::generateConfig(const RtSmithOptions &rtSmithOptions) {
     try {
-        return generateConfigImpl(std::nullopt, rtSmithOptions);
+        return std::move(generateConfigImpl(std::nullopt, rtSmithOptions));
```

Comment:
I do not think this is necessary, std::move on returns rarely is. The compiler usually takes care of that. It is only needed when the object gives up ownership iirc or with unique pointers.

Reply:
True. Fixed. Thanks!